manifest_dev.yml

```diff
@@ -6,6 +6,7 @@ applications:
     disk_quota: 700M
     routes:
       - route: fecfile-web-api-dev.app.cloud.gov
```

Comment:
Can we remove this (and similar for other environments) routes from manifests?

Reply:
@dheitzer yes, I went ahead and swapped them out. Thanks!